doc/tutorial/text_analytics/working_with_text_data.rst

```diff
@@ -57,19 +57,6 @@ the original skeletons intact:
   cp -r skeletons work_directory/sklearn_tut_workspace
 
 
-Machine learning algorithms need data. Go to each ``$TUTORIAL_HOME/data``
-sub-folder and run the ``fetch_data.py`` script from there (after
-having read them first).
-
-For instance:
-
-.. prompt:: bash $
-
-  cd $TUTORIAL_HOME/data/languages
-  less fetch_data.py
-  python fetch_data.py
-
-
```

Comment:
Actually this is still needed for the other datasets. Let me revert this change.